From bd3986daa7ce2882fd1d90e03df2d3ba41fb0890 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Caol=C3=A1n=20McNamara?= <caolanm@redhat.com>
Date: Mon, 13 Mar 2017 12:28:12 +0000
Subject: [PATCH] rhbz#1431476 dnd has started copying instead of moving slides

Is suspect something has changed in gtk3, and I think we're
not supposed to use gdk_drag_drop_succeeded at this point, but
instead we'd get a dragFailed signal on the failure case

Change-Id: I6b305db56a29255b468ca9182d1f7e57cb4c10f4
(cherry picked from commit 4ea76db5e53235971b597a2810a6d12618d6aaea)
---
 vcl/unx/gtk3/gtk3gtkframe.cxx | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/vcl/unx/gtk3/gtk3gtkframe.cxx b/vcl/unx/gtk3/gtk3gtkframe.cxx
index f09905b..979bce0 100644
--- a/vcl/unx/gtk3/gtk3gtkframe.cxx
+++ b/vcl/unx/gtk3/gtk3gtkframe.cxx
@@ -4256,7 +4256,7 @@ void GtkDragSource::dragEnd(GdkDragContext* context)
     {
         datatransfer::dnd::DragSourceDropEvent aEv;
         aEv.DropAction = GdkToVcl(gdk_drag_context_get_selected_action(context));
-        aEv.DropSuccess = gdk_drag_drop_succeeded(context);
+        aEv.DropSuccess = true;
         auto xListener = m_xListener;
         m_xListener.clear();
         xListener->dragDropEnd(aEv);
-- 
2.9.3

